app/components/products_table_component/products_table_component.html.haml

```diff
@@ -0,0 +1,21 @@
+= component_controller(class: "products-table") do
+  .products-table-form
+    .products-table-form_filter_results
+      = render(SearchInputComponent.new(value: @search_term, data: reflex_data_attributes(:search_term)))
+    .products-table-form_categories_selector
+      = render(SuperSelectorComponent.new(title: t("admin.products_page.filters.categories.title"), selected: @categories_selected, items: @categories, data: reflex_data_attributes(:toggle_category), selected_items_i18n_key: "admin.products_page.filters.categories.selected_categories"))
+    .products-table-form_producers_selector
+      = render(SuperSelectorComponent.new(title: t("admin.products_page.filters.producers.title"), selected: @producers_selected, items: @producers, data: reflex_data_attributes(:toggle_producer), selected_items_i18n_key: "admin.products_page.filters.producers.selected_producers"))
+    .products-table-form_per-page_selector
+      = render(SelectorComponent.new(title: "#{@per_page_selected[0]} #{t('admin.products_page.filters.per_page')}", selected: @per_page_selected, items: @per_page, data: reflex_data_attributes(:toggle_per_page)))
```

Comment:
Translations can contain variables which gives translators more flexibility and may be clearer here. In some languages, the grammar may require the number to be after the text.
```suggestion
      = render(SelectorComponent.new(title: t('admin.products_page.filters.per_page', count: @per_page_selected[0]), selected: @per_page_selected, items: @per_page, data: reflex_data_attributes(:toggle_per_page)))
```

You then need to adjust the translation as well, of course.

Reply:
Right! Done ;)